server/src-lib/Hasura/Server/Auth/JWT.hs

```diff
@@ -32,9 +33,9 @@ import           Hasura.Server.Version           (HasVersion)
 
 import qualified Control.Concurrent              as C
 import qualified Crypto.JWT                      as Jose
-import qualified Data.Aeson                      as A
-import qualified Data.Aeson.Casing               as A
-import qualified Data.Aeson.TH                   as A
+import qualified Data.Aeson                      as J
```

Comment:
I thought it would be consistent with the rest of the codebase. When I was reading this file, it felt off. The diff is only in this file, unless someone is working on this file (which I don't think is the case) should it be a problem?

Reply:
I think both `J` and `A` are used in different places (I agree ideally we'd use one or the other). I have two a PR that touches this file, and some local work as well :)

I think best-practices are to:
- carefully weigh the costs of refactors like this: makes the history a little more confusing, can disrupt other work, makes review a little more difficult 
- if you decide to do a refactor try to separate functional changes from refactors (for ease of review and reverting, and more understandable history)

But this is something we have to decide to prioritize as a team, so it's okay with me if you leave this.